pyproject.toml

```diff
@@ -97,22 +97,12 @@ lint.select = [
 ]
 line-length = 108
 lint.ignore = [
-  "D100",    # skip adding docstrings for module
-  "D104",    # ignore missing docstring in public package
-  "D105",    # skip adding docstrings for magic methods
-  "D107",    # skip adding docstrings for __init__
-  "E201",
-  "E202",
-  "E203",
-  "E231",
   "E731",
-  "E702",
-  "S101",
-  "PT004",  # Rule `PT004` is deprecated and will be removed in a future release.
-  "PT013",
-  "PLR0912",
+  "PT004",  # Deprecated and will be removed in a future release.
+  "PLR0912", 
   "PLR0913",
   "PLR0915",
+  "UP017"  # skip `datetime.UTC` alias
```

Comment:
Remove this as project requires python >= 3.11 we can use the `datetime.UTC` alias